# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxFontEntry.h
# Commit: f8ca02252dfc
# Full Hash: f8ca02252dfc48d7342703e6a635d510b9dc0b5d
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-04-02 09:51:45
# Regressor Bug: 1619349
# File Overlap Count: 1
# Description:
#   Bug 1619349 - patch 1 - Record statistics about font-matching behavior in the presContext. r=jwatt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D67787
# ==============================================================================

diff -r 01a2a527aa1a -r f8ca02252dfc gfx/thebes/gfxFontEntry.h
--- a/gfx/thebes/gfxFontEntry.h	Wed Apr 01 21:42:11 2020 +0000
+++ b/gfx/thebes/gfxFontEntry.h	Wed Apr 01 21:42:36 2020 +0000
@@ -770,7 +770,8 @@
   LangPack = 2,  // From an optional OS component such as language support
   User = 3,      // User-installed font (or installed by another app, etc)
   Hidden = 4,    // Internal system font, should never exposed to users
-  Count = 5,     // Count of values, for IPC serialization
+  Webfont = 5,   // Webfont defined by @font-face
+  Count = 6,     // Count of values, for IPC serialization
 };
 
 namespace IPC {
@@ -944,6 +945,9 @@
 
   FontVisibility Visibility() const { return mVisibility; }
   bool IsHidden() const { return Visibility() == FontVisibility::Hidden; }
+  bool IsWebFontFamily() const {
+    return Visibility() == FontVisibility::Webfont;
+  }
 
  protected:
   // Protected destructor, to discourage deletion outside of Release():